_automation/grammars.json

```diff
@@ -340,5 +340,13 @@
     "reference": "v0.5.0",
     "revision": "6129a83eeec7d6070b1c0567ec7ce3509ead607c",
     "updateBasedOn": "tag"
+  },
+  {
+    "language": "zig",
+    "url": "https://github.com/maxxnino/tree-sitter-zig/",
+    "files":["parser.c"],
+    "reference": "master",
```

Comment:
they use `main` instead. If you change it to main and run update with `--force` it works

Reply:
oh i see. nice. will change